scikit-learn-intelex/scripts/download_dataset.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2021 Intel Corporation
+#                    Andrey Morkovkin <andrey.morkovkin@intel.com>
+
+from sklearn.datasets import fetch_openml
+
+def main():
+    X, y = fetch_openml(name='mnist_784', version=1, return_X_y=True, data_home='data', parser='auto')
+    X.to_csv('data/X.csv', index=False)
+    y.to_csv('data/y.csv', index=False)
```

Comment:
@dimakuv 
This would fix warning 
```suggestion
    X.to_csv('data/X.csv', index=False, header=False)
    y.to_csv('data/y.csv', index=False, header=False)
```